.golangci.yml

```diff
@@ -15,3 +15,6 @@ linters:
   - gomnd
   - maligned
   - errcheck
+linters-settings:
+  lll:
+    line-length: 150
```

Comment:
`lll` has 120 as the default and this is what we have configured in other repos. Why do you want to change that?

Suggestion: remove the customization (leave the default) and wrap all changed lines that require it.